docs/usage.md

```diff
@@ -3,130 +3,441 @@
 
-  >**NOTE**: It is recommended to create smaller playbook blocks for easier review and minimal changes in the JSON template generated by FortiAI. You can then combine these small playbook blocks to create a larger playbook for your desired workflow.
+    >*Update the incident with ID [ALERT-ID] with the following details: [Updated details]*
 
-One use case to mitigate threats could require playbook steps that that extract indicators from an alert and enrich them using a threat intelligence tool such as VirusTotal.
+   - The assistant prompts you to provide the updated information for the record.
 
-Based on the reputation, the indicator reputation is updated to *Malicious*, if the reputation returned by VirusTotal is *Suspicious*; or the indicator reputation is updated to *Good* if the reputation is anything but *Suspicious*.
+2. **Provide Updated Information**:
+   - Specify the details to be updated in the record. For example:
 
-You may not be a seasoned playbook developer yet and may require some assistance with developing this playbook.
+    >*Update the alert status to "Resolved" and add a note: [Details of the note]*
 
-Use FortiAI to help with this task by clicking the **Playbook Generation** option in the FortiAI flyout.
+   - The assistant applies the changes to the record based on the provided information.
 
-Type your requirements, such as
+3. **Review and Confirm**:
+   - Review the updated details to ensure accuracy. Confirm the update to finalize the changes to the record.
 
->*Extract Indicators from the Alert Source Data and enrich them using VirusTotal. Based upon reputation, If `malicious` update the indicator reputation to `malicious` else if `suspicious` update the indicator reputation to `suspicious` else update indicator reputation to good*
+### Review Updated Records
 
-Click the **Post** icon. FortiAI, in response, designs the playbook and presents you with the proposed playbook outline:
+- Verify that the record has been updated correctly with the new information. Ensure that all changes are accurate and that the record reflects the most recent data.
 
-![FortiAI - Suggested playbook outline](../docs/res/FortiAIPBGeneration.png)
+### Integrate Updated Records into Workflow
 
-In the review mode itself, you can make modifications, such as changing the integration from VirusTotal to IPStack. Once you are satisfied, click **Ok, Generate Playbook**:
+- Utilize the updated records within your workflow to ensure that your data and analysis reflect the most current information. The updated records can aid in making informed decisions and managing tasks effectively.
 
-![FortiAI - Generate Playbook button](../docs/res/FortiAIPBGenerationReview.png)
+>[!TIP]
+>Regularly review and update records to ensure that all information remains current and accurate. This practice helps maintain the reliability of your data and supports effective decision-making.
```

Comment:
this TIP is also talk the same thing about review

Reply:
Done